### ISSUE TITLE
[Feature Request] Login delay for very first login as well?

### ISSUE THREAD

**Issue by Obihoernchen** [PokemonGoMap/PokemonGo-Map#1513]
I've read that when you're using a delay between each account login, you can run way more accounts on one IP. I don't know whether this is true but some guys at pogodev/apistatus said so.

If you think that's a good idea I'll make a PR for it with docu updates etc.

**Comment by crckdns** [comment]
There is a delay already from what I know, that change was months ago.
Running 127 workers right now from one single IP

**Comment by neskk** [comment]
I used to control the rate of logins using a "sleep 5" between instance launches in my bash script, but now that I'm adding way more workers per instance this doesn't do the trick.
I have gotten away running 200 accounts under one server by always being careful with my launches, of course it can be just pure luck, but I feel that executing 30/40/50+ logins in a matter as fast as possible  - the current way does this - will get your IP flagged more often.
In conclusion, I think this should be addressed and it should actually be quite easy to add a delay between worker logins that can be customized in config.

**Comment by Obihoernchen** [comment]
Btw if somebody want's to change the login delay do it like this:
```diff
diff --git a/pogom/search.py b/pogom/search.py
index d1571a1..04b09d6 100644
--- a/pogom/search.py
+++ b/pogom/search.py
@@ -888,7 +888,7 @@ def check_speed_limit(args, previous_location, next_location, last_scan_time):
 def stagger_thread(args, account):
     if args.accounts.index(account) == 0:
         return  # No need to delay the first one
-    delay = args.accounts.index(account) + ((random.random() - .5) / 2)
+    delay = args.accounts.index(account) * 5 + ((random.random() - .5) / 2)
     log.debug('Delaying thread startup for %.2f seconds', delay)
     time.sleep(delay)
```

**Comment by simonbuehler** [comment]
really needed to set to something around 6 seconds with many accounts on one ip or captchas start to rain
